packages/components/tsconfig.json

```diff
@@ -43,7 +43,6 @@
 		"src/**/*.native.js",
 		"src/**/react-native-*",
 		"src/**/stories/**/*.js", // only exclude js files, tsx files should be checked
-		"src/**/test/**/*.js", // only exclude js files, ts{x} files should be checked
-		"src/index.js"
```

Comment:
On top of excluding `index.js`, I wonder if there's any value in renaming it to `index.ts` ?